gtfs/spec/en/reference.md

```diff
@@ -165,14 +167,14 @@ File: **Required**
 |  `route_text_color` | Color | Optional | Legible color to use for text drawn against a background of `route_color`. Defaults to black (`000000`) when omitted or left empty. The color difference between `route_color` and `route_text_color` should provide sufficient contrast when viewed on a black and white screen. |
 |  `route_sort_order` | Non-negative integer | Optional | Orders the routes in a way which is ideal for presentation to customers. Routes with smaller `route_sort_order` values should be displayed first. |
-|  `continuous_pickup` | Enum | Optional | Indicates that the rider can board the transit vehicle at any point along the vehicle’s travel path as described by `shapes.txt`, on every trip of the route. Valid options are: <br><br>`0` - Continuous stopping pickup. <br>`1` or empty - No continuous stopping pickup. <br>`2` - Must phone agency to arrange continuous stopping pickup. <br>`3` - Must coordinate with driver to arrange continuous stopping pickup.  <br><br>The continuous pickup behavior defined in `routes.txt` can be overridden in `stop_times.txt`. |
-|  `continuous_drop_off` | Enum | Optional | Indicates that the rider can alight from the transit vehicle at any point along the vehicle’s travel path as described by `shapes.txt`, on every trip of the route. Valid options are: <br><br>`0` - Continuous stopping drop off. <br>`1` or empty - No continuous stopping drop off. <br>`2` - Must phone agency to arrange continuous stopping drop off. <br>`3` - Must coordinate with driver to arrange continuous stopping drop off. <br><br>The continuous drop-off behavior defined in `routes.txt` can be overridden in `stop_times.txt`. |
+|  `continuous_pickup` | Enum | Optional | Indicates that the rider can board the transit vehicle at any point along the vehicle’s travel path as described by `shapes.txt`, on every trip of the route. Valid options are: <br><br>`0` - Continuous stopping pickup. <br>`1` or empty - No continuous stopping pickup. <br>`2` - Must phone agency to arrange continuous stopping pickup. <br>`3` - Must coordinate with driver to arrange continuous stopping pickup.  <br><br>Values for `routes.continuous_pickup` will be overriden by values defined in `stop_times.continuous_pickup` for specific `stop_time`s along the route. |
+|  `continuous_drop_off` | Enum | Optional | Indicates that the rider can alight from the transit vehicle at any point along the vehicle’s travel path as described by `shapes.txt`, on every trip of the route. Valid options are: <br><br>`0` - Continuous stopping drop off. <br>`1` or empty - No continuous stopping drop off. <br>`2` - Must phone agency to arrange continuous stopping drop off. <br>`3` - Must coordinate with driver to arrange continuous stopping drop off. <br><br>Values for `routes.continuous_drop_off` will be overriden by values defined in `stop_times.continuous_drop_off` for specific `stop_time`s along the route. |
```

Comment:
2. maybe it should be "may be overridden" instead of "will be overridden"? I understand that if there's a stop_times.continuous_drop_off then it WILL override the value in routes.continuous_drop_off, but the language everywhere else is about the possibilities of the editor, so IMHO "may be" is better.

Reply:
I have to agree that the language/perspective I chose is a bit confusing. I think if when we're speaking of "possibilities of the editor" we should use "can" to denote the ability of doing something. Meanwhile "may" should be used for defining RFC permissions/possibility (i.e., behavior of the spec as a computer would see it).

Reply:
I propose the following:
> Values for < field > can be overridden by defining values in < field >...

Reply:
Please see my suggestion applied in the most recent changes in the "Files changed" tab. Feel free to provide a second review